CMSgov/onemac
# OY2-5832 appendix k waiver

Endpoint: https://d21rvn77q8k58q.cloudfront.net
Story: https://qmacbis.atlassian.net/browse/OY2-5832

### Changes
- Splitting out Appendix K's into it's own form (it used to exist as an option within the Submit new Waiver form)
- Renamed the "Summary" section to "Additional Information" for **all** the form types
- FAQ page updated regarding waiver numbers

### Test
Verify the following:
- Dashboard contains a link under the Waiver section to get to the Appendix K form
- Instructions are visible at the top of the form: “If your Appendix K submission is for more than one waiver number, please just enter one of the applicable waiver numbers. You do not need to create multiple submissions.”
- Verify that a base number of only 4 OR 5 digits can be used (`SS.####.R##.##` OR `SS.#####.R##.##`)
- Verify that the hint text for the waiver number includes `use R00 for waivers without renewals`
- Two possible attachment options are as follows: Appendix K Template (required), Other
- Filling out the form with an id where the _base_ number exists allows successful submission of the form
- Fill out the form with an id where the _base_ number **does not** exist displays a warning message and still allows successful submission of the form
- Can view existing submission via link in the Submissions List on the Dashboard
- Emails are sent to both the submitter and CMS user upon successful submission of App K form with this [Email Content](https://qmacbis.atlassian.net/wiki/spaces/MACPRO/pages/1521647691/OneMAC+Portal+Emails+Previously+SPA+and+Waiver)
- "Summary" section was renamed to "Additional Information" for **all** the request types (shows up on the submit form, view form, and emails)
- FAQ page shows updated content per [FAQ Content](https://qmacbis.atlassian.net/wiki/spaces/MACPRO/pages/1766162435/OneMAC+FAQ+Content)
